lib/diplomat/lock.rb

```diff
@@ -13,15 +13,8 @@ class Lock < Diplomat::RestClient
     # @param value [String] the value for the key
     # @return [Boolean] If the lock was acquired
     def acquire key, session, value=nil
-      raw = @conn.put do |req|
-        url = ["/v1/kv/#{key}"]
-        url += use_named_parameter('acquire', session)
-        url += check_acl_token
-
-        req.url concat_url url
-        req.body = value unless value.nil?
-      end
-      raw.body == 'true'
+      response = raw(:acquire, key, session: session, value: value)
```

Comment:
I like the DRYer nature of this code; however, for readability and consistency I'd like to see the internals of this method stay the same and just add the additional DC parameter.